models.js

```diff
@@ -34,6 +34,20 @@ userSchema.methods.validatePassword = function(password) {
     return bcrypt.compareSync(password, this.Password);
 };
 
+movieSchema.set('toJSON', {
+    transform: function (_, ret) {
+      delete ret.__v
+    }
+  })
+
+
+userSchema.set('toJSON', {
+    transform: function (_, ret) {
+      delete ret.__v
```

Comment:
I know you asked me about the __v this will remove it from the response only when you call res.json(movies or user) other than that it will be there